<launch>
	<rosparam command="load" file="$(find local_planner)/params/cfg.yaml"/>
	<node name="costmap_clear_node" type="costmap_clear_node" pkg="local_planner" output="screen"/>
	<node name="twist_pub_node" pkg="local_planner" type="twist_pub_node" output="screen"/>
</launch>
